clang/test/CodeGen/rtsan_insert_at_exit.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -fsanitize=realtime -emit-llvm -o - %s | FileCheck %s
```

Comment:
this should be merged with `at_entry.c`.

Reply:
Fixed all of your latest round in the latest commit